public/index.html

```diff
@@ -53,7 +53,23 @@
     <noscript>
       You need to enable JavaScript to run this app.
     </noscript>
-    <div id="root"><h3>Loading!</h3></div>
+    <div id="root">
+      <div
+          style="position: absolute; left: 0; top: 0; display: flex; flex-direction: column; height: 100vh; width: 100vw; background-color: #fafbfc; font-family: 'Montserrat', sans-serif; font-variant-numeric: tabular-nums; -webkit-font-smoothing: antialiased; -moz-osx-font-smoothing: grayscale;">
```

Comment:
TOL: `Montserrat` hasn't been requested yet. Does requesting it here do anything (assuming it's not installed as a system level font)?

Reply:
Nope